sashimi/hardware/light_source/interface.py

```diff
@@ -14,34 +15,40 @@ class LaserWarning(Warning):
     pass
 
 
-class AbstractLightSource:
+class AbstractLightSource(ABC):
     def __init__(self, port):
         self.port = port
         self._status = LaserState.OFF
         self.intensity_units = None
 
-    def start(self):
+    def start(self):  #TODO: remove?
         pass
```

Comment:
This method is only initialized in the interface

### Does it make sense to keep? 
my doubt arises from this though:
in the abstract class i would expect 2 types of methods:
 - abstract methods which are `core` methods -> necessary for the implementation of the light source 
 - helper methods which can be used without being overwritten (i.e. sensor resolution property in the camera which doesn't necessarily need to be overwritten each time)
 
 now this method would go in the first category, but it doesn't seem to be necessary for the initialization of the light source -> therefore I think it could be an option to just get rid of it.